erizo_controller/erizoClient/webpack.config.erizo.js

```diff
@@ -7,7 +7,12 @@ module.exports = {
     path: path.resolve(__dirname, 'dist'),
     libraryExport: 'default',
     library: 'Erizo',
-    libraryTarget: 'var',
+    libraryTarget: 'umd',
```

Comment:
This breaks compatibility in some environments, depending on how you import `erizo.js`.  Using var is the most compatible alternative and that's why it's the option we went with.

Reply:
I'm not an expert, @simonemazzoni what do you think?

Reply:
@lodoyun I'm aware that export the lib as UMD could break compatibility in some environments, but it enable the usage of new ES6 import/export functionalities which are standard now. I propose to enhance the webpack build script to produce multiple library target files? Maybe using something like `parallel-webpack`? This allows to import erizo.js code in user code, avoiding to expose it as script in the window (possibly masking it's usage and making easier to handle resource caching and update).

Reply:
@simonemazzoni I understand but I would prefer gradually transitioning to UMD. I do like the `parallel-webpack` variants option to enable the option but keeping it as `'var'` only by default.

We could, for instance, have a configuration file where you could set up an array with different targets. However if that file is not preset, `var` would be the default. That would allow us to transition to var+umd when we're ready to support both.

Reply:
Yes, that solution sounds good to me. Do you have any idea of where to put that config?

Reply:
In erizoClient, where all the build config is. I think is the more sensible place.